### DISCUSSION TITLE
How to add provisioned notifiation channel for grafana in kube-prometheus-stack

### DISCUSSION THREAD

**Discussion by Eljah** [prometheus-operator/kube-prometheus#1271] [Q&A]
Hi!

I'm in stuck with any examples how to set up the grafana notification channel via yaml files on the manner it is done for the custom provisioned dashboards. Currently the dashboard just mention the notification channels ids, so we need the way to set up the notification channels explicitly. Grafana documentation refers this

https://grafana.com/docs/grafana/latest/administration/provisioning/#alert-notification-channels

but it is not clear how to use this for kube-prometheus-stack chart files structureisioning/#alert-notification-channels

**Comment by paulfantom** [comment]
This project is not shipping helm chart.

---

Overall you need to ship additional ConfigMap and mount it in the correct place in grafana container.
